python/amici/petab_import.py

```diff
@@ -185,6 +188,129 @@ def constant_species_to_parameters(sbml_model: 'libsbml.Model') -> List[str]:
     return species_to_parameters(transformables, sbml_model)
 
 
+def import_petab_problem(
+        petab_problem: petab.Problem,
+        folder: str = None,
```

Comment:
stick to `model_output_dir` used below? unnecessarily confusing if the same argument has different names